README.md

```diff
@@ -22,7 +22,8 @@ Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certifie
 -->
 [<img src="docs/static/images/certified-kubernetes-1.23-color.svg" height=150>](https://github.com/cncf/k8s-conformance/pull/2109)
```

Comment:
```suggestion
```

Reply:
Why delete this? We still support 1.23

Reply:
Check the rules on k8s-conformance. We probably cannot post that by the agreement. I used to get email notifications on when the cerifications expire.

Reply:
Ah I see. Updated the PR. Thanks!